src/ERC4973.t.sol

```diff
@@ -180,6 +225,44 @@ contract ERC4973Test is Test, ERC721Holder {
     abt.take(from, bytes(tokenURI), signature);
   }
 
+  function testGiveEmits() public {
+    address from = address(this);
+    address to = address(approver);
+    bytes memory metadata = bytes(tokenURI);
+    bytes32 hash = abt.getHash(from, to, metadata);
+    uint256 tokenId = uint256(hash);
+    bytes memory signature;
+
+    vm.expectEmit(true, true, true, false, address(abt));
+    emit Transfer(address(0), address(this), tokenId);
+    emit Transfer(address(this), to, tokenId);
+    emit Locked(tokenId);
```

Comment:
From my reading of the [docs](https://book.getfoundry.sh/cheatcodes/expect-emit) I don't think this is testing the 3 events but just the first one.